### PR TITLE
Multi scale tta

### PR DESCRIPTION
**Pull request by LucasSloan**
Implements multi-scale tta + wbf's confidence scaling based on the number of boxes.

d0 + multiscale tta gets an AP of 37.1, an improvement of 1.3 over the highest single resolution used when paired with flip tta (35.8)

### CODE REVIEW

File: efficientdet/keras/eval.py [hunk 1]
```diff
@@ -46,51 +47,57 @@ def main(_):
...
   # Network
   model = efficientdet_keras.EfficientDetNet(config=config)
-  model.build((config.batch_size, None, None, 3))
+  model.build((config.batch_size, base_height, base_width, 3))
   model.load_weights(tf.train.latest_checkpoint(FLAGS.model_dir))
 
-  evaluator = coco_metric.EvaluationMetric(filename=config.val_json_file)
+  augmentations = [] 
+  if FLAGS.enable_tta:
+    for size_offset in (0, 128, 256):
```
Comment (mingxingtan) [inline efficientdet/keras/eval.py:60]: Maybe we should use some ratios like [0.5, 0.75, 1.0, 1.25, 1.5]?

File: efficientdet/keras/eval.py [hunk 2]
```diff
@@ -46,51 +47,57 @@ def main(_):
...
+  base_height, base_width = utils.parse_image_size(config['image_size'])
 
   # Network
   model = efficientdet_keras.EfficientDetNet(config=config)
-  model.build((config.batch_size, None, None, 3))
+  model.build((config.batch_size, base_height, base_width, 3))
   model.load_weights(tf.train.latest_checkpoint(FLAGS.model_dir))
 
-  evaluator = coco_metric.EvaluationMetric(filename=config.val_json_file)
+  augmentations = []
```
Comment (mingxingtan) [inline efficientdet/keras/eval.py:58]: Add a comment:  # in format (height, width, flip)

**Comment by mingxingtan** [comment]
Results look great. Thank you, Lucas!

File: efficientdet/keras/eval.py (hunk 1 above)
Comment (LucasSloan) [reply efficientdet/keras/eval.py:60]: I'd like to finetune these more for the larger models, but for d0, it actively hurts to go below the default size (getting rid of that was the major fix I did).  Once I've got this code working internally so I can run the larger models, I'll experiment with these.

File: efficientdet/keras/eval.py (hunk 2 above)
Comment (LucasSloan) [reply efficientdet/keras/eval.py:59]: Done.

**Pull request merged**